docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2384 @@
+**SpectralDimension** objects, each holding a single **SpectralEvent**.  The
+**TransitionQuery** objects select each transition in their respective
+**SpectralDimension** objects.
+
+.. plot::
+    :context: close-figs
+
+    my_mqmas = Method(
+        channels=["87Rb"],
+        magnetic_flux_density=9.4,
+        rotor_frequency = np.inf, # in Hz (here, set to infinity)
+        spectral_dimensions=[
+            SpectralDimension(
+                count=128,
+                spectral_width=6e3,  # in Hz
+                reference_offset=-9e3,  # in Hz
+                label="Symmetric Triple Quantum Frequency",
+                events=[
+                    SpectralEvent(transition_query=[{"ch1": {"P": [-3], "D": [0]}}])
+                ]
+            ),
+            SpectralDimension(
+                count=256,
+                spectral_width=6e3,  # in Hz
+                reference_offset=-5e3,  # in Hz
+                label="Central Transition Frequency",
+                events=[
+                    SpectralEvent(transition_query=[{"ch1": {"P":[-1], "D": [0]}}])
+                ]
+            )
+        ],
+    )
+
+    # Create three sites in RbNO3
+    site1 = Site(
+      isotope="87Rb",
+      isotropic_chemical_shift=-27.4,  # ppm
+      quadrupolar={"Cq":1.68e6, "eta":0.2}  # Cq in Hz
+      )
+    site2 = Site(
+      isotope="87Rb",
+      isotropic_chemical_shift=-28.5,  # ppm
+      quadrupolar={"Cq":1.94e6, "eta":1}  # Cq in Hz
+      )
+    site3 = Site(
+      isotope="87Rb",
+      isotropic_chemical_shift=-31.3,  # ppm
+      quadrupolar={"Cq":1.72e6, "eta":0.5}  # Cq in Hz
+      )
+
+    # No Couplings, so create a separate SpinSystem for each site.
+    sites = [site1, site2, site3]
+    spin_systems = [SpinSystem(sites=[s]) for s in sites]
+
+    sim = Simulator(spin_systems=spin_systems, methods=[my_mqmas])
+    sim.run()
+
+    # Apply Gaussian line broadening along both dimensions
+    processor = sp.SignalProcessor(
+        operations=[
+            sp.IFFT(dim_index=(0, 1)),
+            sp.apodization.Gaussian(FWHM="0.08 kHz", dim_index=0),
+            sp.apodization.Gaussian(FWHM="0.22 kHz", dim_index=1),
+            sp.FFT(dim_index=(0, 1)),
+        ]
+    )
+    dataset = processor.apply_operations(dataset=sim.methods[0].simulation)
+
+
+In the code below, we use the PyPlot method `imshow()
+<https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.imshow.html>`__ to
+return an image of the dataset on a 2D regular raster. We also use `gist_ncar_r
+<https://matplotlib.org/stable/gallery/color/colormap_reference.html>`__ as the
+colormap to map the dataset amplitude to colors. The "_r" indicates a reversed
+version of the "gist_ncar" map. The `colorbar()
+<https://matplotlib.org/stable/api/colorbar_api.html?highlight=colorbar#module-matplotlib.colorbar>`__
+function provides the visualization of the dataset mapping to color to the right
+of the plot.
```

Comment:
@pjgrandinetti This discussion of matplotlib's functions seems a bit extraneous, especially the sentence about the reversal of gist_ncar.

Reply:
Perhaps.  I trying to write for an audience that isn't that familiar with Python.   They're here because they are NMR spectroscopist who want to simulate their own NMR method.   If I recall correctly, this is the first place in both the Introduction and User's Guide that a 2D spectrum is plotted.